internal/services/cosmos/cosmosdb_account_resource_test.go

```diff
@@ -992,7 +992,27 @@ func TestAccCosmosDBAccount_identity(t *testing.T) {
 	})
 }
 
-func TestAccCosmosDBAccount_backup(t *testing.T) {
+func TestAccCosmosDBAccount_icmRegression(t *testing.T) {
+	// Regression test for the Microsoft IcM 466814988...
```

Comment:
Could this test have a more descriptive name? We can also leave out the IcM number comment as these are MSFT-internal.

Reply:
Fixed.